.github/jobs/android.yml

```diff
@@ -9,23 +9,64 @@ jobs:
   timeoutInMinutes: 30
 
   pool:
-    vmImage: macos-13
+    vmImage: macos-latest
 
   steps:
+  - task: JavaToolInstaller@0
+    displayName: 'Set up JDK 17'
+    inputs:
+      versionSpec: '17'
+      jdkArchitectureOption: 'x64'
+      jdkSourceOption: 'PreInstalled'
+
   - script: |
-      echo Install Android image
-      export JAVA_HOME=$JAVA_HOME_8_X64
-      echo 'y' | $ANDROID_HOME/tools/bin/sdkmanager --install 'system-images;android-27;default;x86_64'
-      echo 'y' | $ANDROID_HOME/tools/bin/sdkmanager --licenses
-      echo Create AVD
-      $ANDROID_HOME/tools/bin/avdmanager create avd -n Pixel_API_27 -d pixel -k 'system-images;android-27;default;x86_64'
+      echo "Install NDK and Android SDK"
+      # Use Java 17 which is compatible with modern Android tooling
+      export JAVA_HOME=$JAVA_HOME_17_X64
+      export PATH=$JAVA_HOME/bin:$PATH
+      # Clear any problematic Java options - not needed with Java 17
+      unset _JAVA_OPTIONS
```

Comment:
If it's not needed with Java 17, which it looks we are using, why do we need this?